onnx_tf/backend_rep.py

```diff
@@ -95,15 +91,4 @@ def export_graph(self, path):
 
     :returns: none.
     """
-    graph_proto = self.graph.as_graph_def()
-    # rename the output nodes
-    meaningful_names = {}
-    for output_name in self.outputs:
-      meaningful_names[self.tensor_dict[output_name].name.replace(':0', '')] = output_name
-    for node in graph_proto.node:
-      if node.name in meaningful_names.keys():
-        node.name = meaningful_names[node.name]
-
-    file = open(path, "wb")
-    file.write(graph_proto.SerializeToString())
-    file.close()
+    tf.saved_model.save(self.tf_module, path, signatures=self.tf_concrete_function)
```

Comment:
tf_concrete_function doesn't seem to have been assigned anything. The saved model won't work without input signatures.